src/TurbulenceConvectionUtils.jl

```
@@ -488,7 +488,8 @@ function precondition(
             max_counter = max_counter,
         )
     elseif model_error
-        throw(OverflowError("Number of recursive calls to preconditioner exceeded $(max_counter). Terminating."))
+        @error "Number of recursive calls to preconditioner exceeded $(max_counter). Returning last failed parameter."
```

Is there a reason to throw an `@error` instead of a `@warn`. I guess it doesn't matter as long as the parameters are returned.

---

Not particularly. I have just been using @warn more loosely, and I keep @error for things that should require close inspection.